tests/phpunit/integration/Modules/Analytics_4/Resource_Data_Availability_DateTest.php

```diff
@@ -91,6 +103,8 @@ public function test_reset_resource_date() {
 	}
 
 	public function test_get_all_resource_dates() {
+		$this->enable_feature( 'audienceSegmentation' );
```

Comment:
The `audienceSegmentation` feature flag has been removed and doesn't need enabling here.